zilencer/management/commands/populate_db.py

```diff
@@ -287,7 +287,7 @@ def restore_saved_messages():
     client_set = set(["populate_db", "website", "zephyr_mirror"])
     huddle_user_set = set() # type: Set[Tuple[text_type, ...]]
     # First, determine all the objects our messages will need.
-    print(datetime.datetime.now(), "Creating realms/streams/etc...")
+    print(now(), "Creating realms/streams/etc...")
     def process_line(line):
```

Comment:
This is printing to console, so the original local time was probably intended behavior, and can be left.